client/state/selectors/is-sharing-buttons-save-successful.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Returns true if the save sharing buttons requests is successful
+ *
+ * @param  {Object}  state  Global state tree
+ * @param  {Number}  siteId Site ID
+ * @return {Boolean}        Whether the request is successful or not
+ */
+export default function isSiteSettingsSaveSuccessful( state, siteId ) {
```

Comment:
Thanks for the clarification - it seems sometimes we are "okay" with very targeted selectors, when other times there is push back on adding selectors